L/libcgal_julia/build_tarballs.jl

```diff
@@ -3,61 +3,78 @@
 using BinaryBuilder
 import Pkg: PackageSpec
 
-const name = "libcgal_julia"
-const version = v"0.14"
+name = "libcgal_julia"
+version = v"0.14"
+
+isazure = haskey(ENV, "BUILD_BUILDNUMBER")
```

Comment:
Now we have the `YGGDRASIL` environment variable here, to identify our CI pipeline

Reply:
Awesome! I'll switch over right away 👍